source/cursorManager.py

```diff
@@ -26,9 +25,13 @@
 import braille
 import vision
 import controlTypes
-from inputCore import SCRCAT_BROWSEMODE
+from inputCore import (
+	SCRCAT_BROWSEMODE,

```

Comment:
another category should be created for document navigation

Reply:
Sure, but I need lots of help. I still don't know what these categories do for scripts which cannot be reassigned. The assignment in this case appears to apply to the entire class. Should a new category be assigned to just the paragraph navigation scripts? If so, where should this constant be defined? All such constants seem to be defined randomly in various modules at the moment.
In the end, I still don't see what this gains us.

Reply:
Sure, the scripts to jump to next/previous paragraph have no description so that they cannot be reassigned, which is a good thing.

Thus IMO changing the category in this PR is not necessary and is a bit off-topic as I wrote before. We can always change the category in a subsequent PR/work.

@seanbudd do you still request this category to be changed? For which purpose? I do not understand either.

Reply:
If they cannot be reassigned, there is not much purpose in changing the category.
If they are accessible via the input gestures dialog, it would be ideal if they were in their own category for document navigation.

Reply:
The scripts to jump to next/previous paragraph have no description so they are not accessible in input gesture dialog.
Sorry for my misleading comment at the beginning of this discussion.